app/browser/api/ledger.js

```diff
@@ -1562,6 +1575,7 @@ const onWalletProperties = (state, body) => {
   const balance = parseFloat(body.get('balance'))
   if (balance >= 0) {
     state = ledgerState.setInfoProp(state, 'balance', balance)
+    lockInContributionAmount()
```

Comment:
shouldn't `balance` be passed as the parameter?

Reply:
You are absolutely right! doh (great catch!)